<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysUserMgrPage</class>
 <widget class="QWidget" name="SysUserMgrPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QPushButton" name="delButton">
     <property name="text">
      <string>删除用户</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="8">
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>用户管理</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>171</number>
        </attribute>
        <column>
         <property name="text">
          <string>用户名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>用户编号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>所属组</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>登陆终端</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="modButton">
     <property name="text">
      <string>编辑用户</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="freshButton">
     <property name="text">
      <string>刷新列表</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>添加用户</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="logOutButton">
     <property name="text">
      <string>断开终端</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
